VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+                           <incipit xml:lang="gez">
+                              <locus target="#159va"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘሠሉስ፡</hi> አክሊለ፡ ምክህነ፡ ወቀዳሚተ፡ መድኃ<supplied reason="omitted" resp="MV">ኒ</supplied>ትነ። ወመሠረተ፡ ንጽሕነ፡ 
+                              ኮነ፡ በ<hi rend="rubric">ማርያም፡</hi> ድንግል። እንተ፡ ወለደት፡ ለነ፡ ዘእ<hi rend="ligature">ግዚ</hi>አብሔር፡ ቃለ።
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#161vb"/>
+                              ይሰፍሑ፡ ክነፊሆሙ። ወይብሉ፡ ዝንቱ፡ ውእቱ፡ ንጉሠ፡ <choice><sic>ሰ</sic><corr resp="MV">ስ</corr></choice>ብሐት፡ መጽአ፡ ይሥረይ፡ ኃጢአተ፡ ዓለም፡ 
+                              በዕበየ፡ ሣህሉ። ሰ፡ ቅ፡ 
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.3">
+                           <locus from="161vb" to="164ra"/>
+                           <title type="complete" ref="LIT2509Weddas#Wednesday"/>
+                           <incipit xml:lang="gez">
+                              <locus from="161vb" to="162ra"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘረቡዕ፡</hi> ኵሉ፡ ሠራዊተ፡ ሰማያት፡ ይብሉ፡ ብፅዕት፡ ሰማይ፡ ዳግሚት፡ ዲበ፡ ምድር። ኆኅተ፡ ምሥራቅ፡ 
+                              <hi rend="rubric">ማርያም፡</hi> ድንግል፡ ከብካብ፡ ንጹ<pb n="162r"/><cb n="a"/>ሕ፡
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#164ra"/>
+                              ወአንሶሰወ፡ ምስለ፡ ሰብእ፡ እስመ፡ መሐረ፡ ውእቱ፡ ወመፍቀሬ፡ ሰብእ፡ አድኃነ፡ ነፍሳቲነ፡ በምጽአቱ፡ ቅዱስ፡ ሰ፡ ቅ።
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.4">
+                           <locus from="164ra" to="166rb"/>
+                           <title type="complete" ref="LIT2509Weddas#Thursday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#164ra"/>
+                              <hi rend="rubric">ውዳሴ፡ ዘሐሙስ፡ ዕፀ፡ እንተ፡</hi> ርእየ፡ ሙሴ፡ በነደ፡ እሳት፡ ውስተ፡ ገዳም፡ ወአዕፁቂሃ፡ ኢትውኢ፡ ትመስል፡ <hi rend="rubric">ማርያም፡</hi> 
+                              ድንግል፡ ዘእንበለ፡ ርኩስ።
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#166rb"/>
+                              ወተወልደ፡ በቤተ፡ ልሔም፡ በከመ፡ ሰበኩ፡ ነቢያት፡ አድኃነነ፡ ወቤዘወነ፡ ወረሰየነ፡ ሕዝበ፡ ዚአሁ፡ ሰ፡ ቅ።
+                           </explicit>
+                        </msItem>
+                        <msItem xml:id="ms_i1.5.5">
+                           <locus from="166rb" to="167vb"/>
+                           <title type="complete" ref="LIT2509Weddas#Friday"/>
+                           <incipit xml:lang="gez">
+                              <locus target="#166rb"/>
+                              <hi rend="rubric"><surplus>ው</surplus>ው<add place="interlinear">ዳ</add>ሴ፡ </hi> ዘዓርብ፡ ቡርክት፡ አንቲ፡ እምአንስት፡ ወቡሩክ፡ ፍሬ፡ 
```

Comment:
Forse la spazio tra ሴ፡ e </hi> è da eliminare? (non so se influisce sul layout del testo)

Reply:
For Sanne: I was asking if the space between  ሴ፡  and </hi> shall be deleted

Reply:
I might not be relevant. I've removed the space anyway

Reply:
Alright!